setup.cfg

```diff
@@ -29,12 +29,12 @@ classifiers =
 packages = find:
 install_requires =
     cftime>=1.2.1
-    netcdf4
-    numpy>=1.15
+    netcdf4<1.6.1
```

Comment:
This pin will hurt us when trying to install pocean in modern envs.
I was unable to fix the failing tests without it. [Iris wrote a whole new class](https://github.com/SciTools/iris/blob/main/lib/iris/fileformats/netcdf/_thread_safe_nc.py) to solve that. I was hoping for an easier solution here.